docs/docs/Releases/release120.md

```diff
@@ -0,0 +1,108 @@
+# Release 1.2.0
+
+Date: 15th Dec, 2022
+
+## Overview
+
+Klaw version 1.2.0 is a minor release that includes a number of
+improvements and new features designed to enhance the Klaw user
+experience.
+
+Some of the key features and enhancements in this release include the
+introduction of React UI, a more user-friendly and intuitive interface
+for browsing and working with topics in Klaw, support for Azure AD
+authentication, enhancements to the codebase, and more.
+
+## What's new in Klaw 1.2.0
+
+### Introducing React for UI
+
+React is a popular open-source JavaScript library that simplifies the
+development of front-end applications and provides a great alternative
+to large, complex front-end frameworks like AngularJS. Klaw's current
+user interface is built with AngularJS, which is no longer officially
+supported by Google. As an older framework, it does not leverage modern
+or component-based development.
+
+React, on the other hand, provides greater flexibility and modularity,
+making it easier to design reusable components and scale an application.
+Furthermore, React makes it easier to create consistent and cohesive
+user experiences.
+
+Moving to React for Klaw's UI provides numerous benefits in terms of
+performance, scalability, ease of development, and support by a large
+community of developers. We've tested React with our current interface,
+and it looks amazing - we're excited to start this new journey.
+
+### New look and feel for Browse topics
+
+With the switch to React for UI, this release also introduces the
+redesigned user interface for Browse topics, providing a more
+user-friendly and intuitive interface for managing Kafka topics in Klaw.
+It consists of changes to the layout, styling, and overall appearance of
+the interface, as well as improvements to make browsing and working
+topics easier, more efficient, and more accessible.
+
+To preview the new Klaw user user interface, open the
+`application.properties` file on the Klaw **core** module, and set the
+value of the following property to true: :: \# Enable new Klaw user
```

Comment:
The following should be on a new line I missed this the first time I'm sorry, but the note looks perfect now.
# Enable new Klaw user interface 
klaw.coral.enabled=true